galera/src/certification.cpp

```diff
@@ -185,10 +185,10 @@ check_against(const galera::KeyEntryNG*   const found,
                          << " for key " << key << ": "
                          << *trx << " <---> " << *ref_trx;
             }
-            /* fall through */
+            __attribute__((fallthrough));
```

Comment:
Here and everywhere. Wouldn't it be better to use C++17 `[[fallthrough]]` attribute? 

Currently, in all other places we are using [[fallthrough]] attribute since https://github.com/percona/percona-server/commit/c5923aa1aedea4b4268e2b372098e2bf

Reply:
Galera sets CMAKE_CXX_STANDARD to 11.